File: 1672-richest-customer-wealth/1672-richest-customer-wealth.py
class Solution:
    def maximumWealth(self, accounts: List[List[int]]) -> int:
        rows = len(accounts)
        cols = len(accounts[0])

        wealth = 0
        for r in range(rows):
            customer_sum = 0
            for c in range(cols):
                customer_sum += accounts[r][c]

            wealth = max(wealth, customer_sum)

        return wealth  

File: 1672-richest-customer-wealth/1672-richest-customer-wealth.java
class Solution {
    public int maximumWealth(int[][] accounts) {
        int maxWealth = 0;

        for (int r = 0; r < accounts.length; r++) {
            int curWealth = 0;
            for (int c = 0; c < accounts[r].length; c++) {
                curWealth += accounts[r][c];
            }

            if (curWealth > maxWealth) {
                maxWealth = curWealth;
            }
        }
        return maxWealth;
    }
}